src/it/sample-plugin/pom.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
Not sure what you are proposing exactly.

If there are particular POM idioms which are thought to be problematic, or which have had known regressions, then we can add ITs for those later. For now I just kicking things off with a smoke test.

In no case do I wish to check out a _branch_ of some other repository, because that makes the build here nondeterministic.